src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -149,6 +149,345 @@ Once the server is up, ProActive Web Portals are available at `\http://localhost
 
 Your ProActive Scheduler is ready to execute <<_glossary_job,*Jobs*>>!
 
+
+==== How to install ProActive using Docker Containers
+ProActive could be easily deployed and started within a fully containerized environment. For this purpose, we provide a ready-to-use Docker Compose configuration that enables you running ProActive Workflows and Scheduling as a containerized service.
+It starts the following services:
+
+*  Proactive server container
+*  Proactive node container
+*  Proactive database container (Postgres or Mysql). The database container is not started when using our embedded HSQL database.
+
+The Docker Compose configuration pulls the Proactive server, node, and database images from the https://dockerhub.activeeon.com/[Private Activeeon Docker Registry^].
+
+These images can be used out of the box, on any operating system. You can customize the server, nodes, and database settings, located in the `.env` file according to your requirements.
+
+To be able to access the https://dockerhub.activeeon.com/[Private Activeeon Docker Registry^], you need to have credentials.
+Please contact us at contact@activeeon.com to request access.
+
+In the following, we describe the installation steps you need to follow in order to have Proactive running in a fully containerized environment.
+
+You need first to create and configure the Docker compose configuration file `.env` to include the following content:
+
+===== Docker compose configuration file: `.env`
+
+[source]
+----
+# Public Host IP Address
+HOST_ADDRESS=XX.XX.XX.XX
+# Turn it on if the host OS is Mac
+HOST_OS_MAC=off
+# Release version, it could be 'trial' or 'enterprise'
+RELEASE=enterprise
+# Proactive version to be deployed
+PROACTIVE_VERSION=11.1.0
+# Protocol to use for accessing Web apps deployed by the proactive server
+PROTOCOL=https
+# Port to use for accessing Web apps deployed by the proactive server
+PORT=8443
+# Remove history older than (in days), 30 days by default
+JOB_CLEANUP_DAYS=30
+# User starting the ProActive service
+UID=1000
+GID=1000
+USER_NAME=activeeon
+GROUP_NAME=activeeon
+# Required to run docker in docker on Linux, look at /etc/group or run 'getent group docker | cut -d: -f3' to get the host docker gid
+DOCKER_GID=115
+# Number of ProActive nodes to start on the server machine
+PROACTIVE_WORKER_NODES=7
+# Proactive admin password
+PROACTIVE_ADMIN_PASSWORD=activeeon
+# External proactive db credentials, used only for postgres and mysql compose configurations
+POSTGRES_PASSWORD=change_me
+MYSQL_ROOT_PASSWORD=change_me
+DB_RM_PASS=change_me
+DB_SCHEDULER_PASS=change_me
+DB_CATALOG_PASS=change_me
+DB_PCA_PASS=change_me
+DB_NOTIFICATION_PASS=change_me
+
+----
+
+Once, you create and configure the `.env` file, you can select one of the three following Docker Compose configurations to start the Proactive server and node containers.
+
+For instance, we provided support to three database systems, namely HSQLDB, Postgres, and MySQL.
```

Comment:
Do you mean, "We currently provide support to "?

Reply:
yes, I will fix it